### PR TITLE
Makes station traits config option

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Station traits are config option.
Note to anyone who merges it - they are supposed to be enabled on live servers while disabled in the repo so they are not rolled while debugging.

## Why It's Good For The Game

requested by ike

## Changelog
:cl:
add: Station traits are config option.
/:cl:

### CODE REVIEW

**Review by francinum — APPROVED** [review]

File: config/Sage/config.txt [hunk 1]
```diff
@@ -551,3 +551,6 @@ VOTE_AUTOTRANSFER_INTERVAL 18000
 
 ## Ghost role cooldown time after death (In deciseconds)
 GHOST_ROLE_COOLDOWN 3000
+
+## Enable/disable roundstart station traits
+#STATION_TRAITS
```
Comment (ike709) [inline config/Sage/config.txt:556]: These should be enabled by default to match the server config.

Comment (Archanial) [reply config/Sage/config.txt:556]: This is similar to #3766
It is supposed to be disabled in repo so people don't get traits while debugging stuff but enabled on the live server

Comment (PowerfulBacon) [reply config/Sage/config.txt:556]: It should be disabled on the repo so that dev builds don't use station traits.

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
Config requires updating (Needs them enabled)

**Pull request merged**